### ISSUE TITLE
Explain Cascading Renderers

### ISSUE THREAD

**Issue by nleanba** [rdf2h/rdf2h-documentation#1]
at selecting renderer ?

**Comment by retog** [comment]
Good point. This probably requires changing the documentation tool to support multiple renderer graphs. Actually it might already be possible, just by adding an editor with the respective class
